### PR TITLE
rename xdot in python.yaml to python-xdot, becase xdot is already tak…

### PR DESCRIPTION
**Pull request by k-okada**
…en by ros package, atleast until indigo http://wiki.ros.org/xdot

rewrited version of #15257

some of rosdep keys are multiply defined, and that sometime confuse users.

rename xdot in python.yaml to python-xdot, becase xdot is already taken by ros package, atleast until indigo http://wiki.ros.org/xdot
change xdot to python-xdot, this will break backword compatibility....

### CODE REVIEW

**Comment by dirk-thomas** [comment]
This needs to be rebased.

**Comment by k-okada** [comment]
rebased

**Review by tfoote — APPROVED** [review]
lgtm, though before we merge this @k-okada could you announce this and the proposed pocketsphinx change to General on Discourse?

**Comment by k-okada** [comment]
@tfoote ok, i have posted announcement https://discourse.ros.org/t/proposed-rename-of-rosdep-keys-pocketsphinx-and-xdot/1959/2 , or are you inted to create new post for announce?
I also download all released packages with `rosinstall_generator ALL --rosdistro kinetic` and `rosinstall_generator ALL --rosdistro package` and grep-ed to find keys, also googled and created tickets for those who uses renamed keys.

**Comment by tfoote** [comment]
Thanks @k-okada the post and grepping for the keys is about all we can do.

@ros/rosdistro-maintainers all set for merge.

**Review by dirk-thomas — CHANGES_REQUESTED** [review]
> grepping for the keys is about all we can do

I am not sure if any of you has actually done this but currently at least two released ROS packages use this key. Therefore I don't think this should be merge until after PRs for these packages have been proposed to notify the maintainers. As far as I can see it affects:

* [ecto](https://github.com/plasmodic/ecto/blob/30e50710ed4c39428f5e534363a27239ca5f8a7b/package.xml#L17)
* [srv_tools](https://github.com/srv/srv_tools/blob/ccafc03236e6ea80f67ce523216ab0729727a244/launch_tools/package.xml#L18)

**Comment by k-okada** [comment]
> I am not sure if any of you has actually done this

I did and had same results as you did. I have already filed a ticket to thier repository and one of them respond already, so I can take care of them.

https://github.com/srv/srv_tools/issues/16
https://github.com/plasmodic/ecto/issues/297

**Comment by k-okada** [comment]
@tfoote @dirk-thomas I think I have satisfied your change request, please check again

**Review by dirk-thomas — APPROVED** [review]

**Pull request merged**
